showcase/sql-practice.md

```diff
@@ -0,0 +1,17 @@
+---
+name: sql-practice
+logo_url: https://www.sqlite.org/images/sqlite370_banner.gif
+website: https://www.sql-practice.com
+description: A practical example of sql.js being implemented on a site.
+keywords: language javascript vue sql
+---
+
+Sql-practice is a site designed to help people practice and learn sql by solving sql query problems.
+
+![sql-practice screenshot](https://i.imgur.com/YOCFVxd.png)
+
+sql-practice was written to use sql.js in a practical use. Implementing this site prior to sql.js would have required a lot of server cost but now it can be done on a static site.
```

Comment:
Could you mention here that: ["sql.js uses emscripten to compile SQLite to webassembly "](https://sql.js.org/#/). 😄

Reply:
I believe I made the change. 

Thank you for maintaining the site by the way. It helped me get inspiration for other wasm-based sites.